virttest/utils_libvirt/libvirt_usb_utils.py

```diff
@@ -0,0 +1,42 @@
+"""
```

Comment:
For the file name, let's align most files's style in this directory and make it shorter, like 'libvirt_usb.py', ok?

Reply:
+1

Reply:
@dzhengfy 
@Yingshun 
updated